src/components/study-pane.js

```diff
@@ -393,6 +403,10 @@ const StudyPane = (props) => {
 
     const updateLoadFlowResult = useCallback(() => {
         fetchStudy(studyUuid).then((study) => {
+            if (study.status === 400 || study.status === 404) {
+                setStudyNotFound(true);
```

Comment:
Is it ok to check if the study exists here only ?
The LF infos could be moved and we lose the check on study existance.